Pipfile

```diff
@@ -0,0 +1,18 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+"discord.py" = "==1.6.0"
+flake8 = "==3.9.0"
+praw = "==6.2.0"
+prawcore = "==1.0.1"
+pytest = "==6.2.3"
+pytest-cov = "==2.11.1"
+redis = "==3.5.3"
+
+[dev-packages]
+
+[requires]
+python_version = "3.8"
```

Comment:
As of right now, this supports Python 3.6+. Also, the python_version is an optional parameter in the pipfile. Personally, I think this should be bumped down to 3.6, and the CICD can build on 3.6, 3.7, and 3.8 to ensure compatibility (at least where we can).

Reply:
ok